magento/devdocs#: Get Redis version on Cloud environments (#8016)
magento/devdocs: src/cloud/project/services-redis.md

   ```bash
   redis-cli -h redis.internal
   ```

## Get installed Redis version

> Use the following command to get a Redis version on the Integration environment:

Reviewer: ```suggestion
Use the following command to get the Redis version installed in an Integration environment:
```

Author: **done**

Author: Thank you, @meker12

Reviewer 2: Normally we'd say "ON an Integration environment"

Reviewer 2: Same goes for above on the Staging & Production

Author: **done**

@BarnyShergold , could you please review?

Thank you!

```bash
redis-cli -h redis.internal info | grep version
```

```terminal
redis_version:5.0.0
gcc_version:6.3.0
```

> To get a Redis version on the Staging and Production environments use `redis-server` command:

Reviewer: ```suggestion
To get the Redis version installed in a Staging or Production environment, use the `redis-server` command:
```

Author: **done**
